rust/src/dictionary.rs

```diff
@@ -57,7 +58,7 @@ impl Dictionary {
     }
 
     pub fn size(&self) -> usize {
-        unsafe { root::keyvi_dictionary_get_size(self.dict) }
+        unsafe { root::keyvi_dictionary_get_size(self.dict) as usize }
```

Comment:
Is there a reason why this is an `as` cast as opposed to the `try_into`s below?

Reply:
No, not really. I can change into `try_into` here to be consistent with the rest, but I'm thinking now to change `size(()` functions bellow in the `vector.rs` to use `as` cast instead of `try_into`. I think I prefer it to `try_into` there because it's more readable/explicit. Do you know is there any advantage to using  `try_into` instead?

Reply:
If the cast can "fail" and you want to notice it, you should use `try_into`, otherwise `as` is fine.
e.g.

```
use std::convert::TryInto;

fn main() {
    println!("u64: {}", u64::max_value());
    println!("u64 as u32: {}", u64::max_value() as u32);
    let x: Result<u32, _> = u64::max_value().try_into();
    println!("u64 try_into() u32: {:?}", x);
}
```
Prints:
```
u64: 18446744073709551615
u64 as u32: 4294967295
u64 try_into() u32: Err(TryFromIntError(()))
```

In this case it shouldn't really matter since for us `usize` will always be the same size as `u64`.

Reply:
Yes, I don't think it matters in practice, but let's use `try_into` anyway.